docs/sample-v2-addon/rollup.config.mjs

```diff
@@ -0,0 +1,60 @@
+import { Addon } from '@embroider/addon-dev/rollup';
+import postcss from 'rollup-plugin-postcss';
+import typescript from 'rollup-plugin-ts';
+
+const addon = new Addon({
+  srcDir: 'src',
+  destDir: 'dist',
+});
+
+export default {
+  // This provides defaults that work well alongside `publicEntrypoints` below.
+  // You can augment this if you need to.
+  output: addon.output(),
+
+  plugins: [
+    // These are the modules that users should be able to import from your
+    // addon. Anything not listed here may get optimized away.
+    addon.publicEntrypoints([
+      'components/**/!(*.css.d).js',
+      'index.js',
+      'template-registry.js',
+    ]),
+
+    // These are the modules that should get reexported into the traditional
+    // "app" tree. Things in here should also be in publicEntrypoints above, but
+    // not everything in publicEntrypoints necessarily needs to go here.
+    addon.appReexports(['components/**/!(*.css.d).js']),
```

Comment:
When `type-css-modules` is used to create the declaration files `*.css.d.ts`, we want to exclude them from `publicEntrypoints` and `appReexports` (assuming that the addon maintainer does want the end-developers _not_ to refer to the addon's stylesheets).

I found two ways to exclude the files: The addon maintainer can either (1) explicitly list the correct files (i.e. the maintainer has to update the list when there is a new component, etc.) or (2) use a glob pattern to include `*.js` files but not `*.css.d.js`.